ClientCore/Settings/UserINISettings.cs

```diff
@@ -332,13 +354,11 @@ public void SaveSettings()
         }
 
         public bool IsGameFiltersApplied()
-        {
-            return ShowFriendGamesOnly.Value != DEFAULT_SHOW_FRIENDS_ONLY_GAMES ||
-                   HideLockedGames.Value != DEFAULT_HIDE_LOCKED_GAMES ||
-                   HidePasswordedGames.Value != DEFAULT_HIDE_PASSWORDED_GAMES ||
-                   HideIncompatibleGames.Value != DEFAULT_HIDE_INCOMPATIBLE_GAMES ||
-                   MaxPlayerCount.Value != DEFAULT_MAX_PLAYER_COUNT;
-        }
+            => ShowFriendGamesOnly.Value != DEFAULT_SHOW_FRIENDS_ONLY_GAMES ||
+               HideLockedGames.Value != DEFAULT_HIDE_LOCKED_GAMES ||
+               HidePasswordedGames.Value != DEFAULT_HIDE_PASSWORDED_GAMES ||
+               HideIncompatibleGames.Value != DEFAULT_HIDE_INCOMPATIBLE_GAMES ||
+               MaxPlayerCount.Value != DEFAULT_MAX_PLAYER_COUNT;
```

Comment:
operators should be at beginnings of the lines, not the end. we read from left to right so it's easier to spot them this way

Reply:
Done by 9b425f1cc90370a4a651c1d28104b4eaa1e3b53c.